instrumentation/opentelemetry-instrumentation-falcon/tests/test_falcon.py

```diff
@@ -264,3 +265,18 @@ def test_hooks(self):
         self.assertEqual(
             span.attributes["request_hook_attr"], "value from hook"
         )
+
+
+class TestFalconInstrumentationWrappedWithOtherFramework(TestFalconBase):
```

Comment:
Why are changes unrelated to the issue are shown here?

Reply:
I fetched and commit the remote repo to my forked repo and pull main branch to my develop branch.
then I pushed my changes. So i guess along with my changes those changes were also shown here.

Reply:
@sanketmehta28 can you please rebase on latest main branch and update this PR?